src/intellisense/languageServer.node.ts

```diff
@@ -79,6 +91,8 @@ class NerfedExecuteCommandFeature implements DynamicFeature<ExecuteCommandRegist
     }
 }
 
+let languageClient: LanguageClient | undefined;
```

Comment:
This can't be a global variable. The LanguageServer is not a singleton.

Reply:
ah.. okay.